biohansel/cli.py

```diff
@@ -247,35 +251,91 @@ def parse_comma_delimited_floats(ctx: click.Context, param: click.Option, value:
 
 
 @cli.command()
-@click.option('--vcf-file-path',
+@click.option('-v', '--vcf-file-path',
               required=True,
               type=click.Path(exists=True, dir_okay=False),
               help='Variant calling file (VCF) of a collection of input genomes for population of interest against a '
                    'reference genome that must be specified with --reference-genome-path')
-@click.option('--reference-genome-path',
+@click.option('-r', '--reference-genome-path',
               required=True,
               type=click.Path(exists=True, dir_okay=False),
               help='Reference genome assembly file path. The reference used in the creation of the input VCF file.')
 @click.option('--phylo-tree-path',
               required=False,
               type=click.Path(exists=True, dir_okay=False),
               help='Optional phylogenetic tree created from variant calling analysis.')
-@click.option('--distance-thresholds',
+@click.option('-d', '--distance-thresholds',
               required=False,
               type=str,
               callback=parse_comma_delimited_floats,
               help='Comma delimited list of distance thresholds for creating hierarchical clustering groups '
                    '(e.g. "0,0.05,0.1,0.15")')
-def create(vcf_file_path, reference_genome_path, phylo_tree_path, distance_thresholds):
+@click.option('-o', '--output-folder-name',
+              required=True,
+              type=click.Path(exists=False, dir_okay=True),
+              help='Output folder name in which schema file would be located'
+              )
+@click.option('-s', '--schema-name',
+              required=False,
+              default="biohansel-schema",
+              type=str,
+              help='A unique name for the schema file that is generated, the default is just'
+                   '{bio_hansel-schema}-reference_genome_name}-{schema_version}')
+@click.option('-m', '--schema-version',
+              required=False,
+              default="0.1.0",
+              type=str,
+              help='An optional version number for the schema file that is generated')
+@click.option('-t', '--tile-length',
+              required=True,
+              type=int,
+              help='Length of sequence to be added around each SNV in the schema file,'
+              'if an even integer is provided, then the tile length would be n+1'
+              )
+@click.option('-f', '--reference-genome-format',
+              required=True,
+              type=click.Choice(['fasta', 'genbank']),
+              help='Reference genome file format: can be either fasta or genbank format'
+              )
+@click.option('-g', '--min-group-size',
+              type=click.Choice(['2', '3', '4', '5', '6', '7', '8', '9', '10']),
```

Comment:
Please see [my comment](https://github.com/phac-nml/biohansel/pull/58#discussion_r209313722) about making this option accepting a range of integer values. 

> Let's also set a default for this value to reduce the amount of things users need to specify via the command-line and that make sense like integers between 2 and 10 (or half of number of input samples) inclusive.

e.g. `hansel create ... -g 2-50` -> creates schemes with min group size from 2 to 50 inclusive.

Reply:
ok yes this has been addressed in d141a97fd0744cfd2f291e96d02ced7710935307